misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/LocalDateFeatureAssertionsSpec.kt

```diff
@@ -11,23 +11,28 @@ import java.time.LocalDate
 abstract class LocalDateFeatureAssertionsSpec(
     yearFeature: Feature0<LocalDate, Int>,
     year: Fun1<LocalDate, Expect<Int>.() -> Unit>,
+    monthFeature: Feature0<LocalDate, Int>,
+    month: Fun1<LocalDate, Expect<Int>.() -> Unit>,
     describePrefix: String = "[Atrium] "
 ) : Spek({
 
     include(object : SubjectLessSpec<LocalDate>(describePrefix,
         yearFeature.forSubjectLess().adjustName { "$it feature" },
-        year.forSubjectLess { isGreaterThan(2000) }
+        year.forSubjectLess { isGreaterThan(2000) },
+        monthFeature.forSubjectLess().adjustName { "$it feature" },
+        month.forSubjectLess { isLessThan(12) }
     ) {})
 
     include(object : AssertionCreatorSpec<LocalDate>(
         describePrefix, LocalDate.ofYearDay(2040, 1),
-        year.forAssertionCreatorSpec("$toBeDescr: 1") { toBe(2040) }
+        year.forAssertionCreatorSpec("$toBeDescr: 1") { toBe(2040) },
+        month.forAssertionCreatorSpec("$toBeDescr: 1") {toBe(1)}
     ) {})
 
     fun describeFun(vararg funName: String, body: Suite.() -> Unit) =
         describeFunTemplate(describePrefix, funName, body = body)
 
-    val fluent = expect(LocalDate.ofYearDay(2009, 40))
+    val fluent = expect(LocalDate.now().withMonth(3).withYear(2009))
```

Comment:
I suggest to use the following, is more readable IMO
```suggestion
    val fluent = expect(LocalDate.of(2009, Month.MARCH, 13))
```
requires `import java.time.Month` in addition.